pkg/ddevapp/ddevapp.go

```diff
@@ -1423,7 +1423,7 @@ Fix with 'ddev config global --required-docker-compose-version="" --use-docker-c
 	if !nodeps.ArrayContainsString(app.GetOmittedContainers(), "db") {
 		dependers = append(dependers, "db")
 	}
-	output.UserOut.Printf("Waiting for web/db containers to become ready: %v", dependers)
+	output.UserOut.Printf("Waiting for containers to become ready: %v", dependers)
```

Comment:
I think this is fine. The reason this was stated the way it was is that we do two waits, the first for core containers, and the second for anything else.